src/styles.scss

```diff
@@ -1 +1,2 @@
 /* You can add global styles to this file, and also import other style files */
+@import "~leaflet/dist/leaflet.css";
```

Comment:
Should this be in `world-map-component.scss` instead?

Reply:
this is a file from the leaflet package, can we rename it?

Reply:
No, but you can add the import line inside `world-map-component.scss` instead of here.
Since we're only using leaflet for the map it makes sense to only use the styling for the component and not globally, I think.